Signal/src/util/UI Categories/UIResponder+OWS.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  Copyright (c) 2018 Open Whisper Systems. All rights reserved.
+//
+
+// Based on https://stackoverflow.com/questions/1823317/get-the-current-first-responder-without-using-a-private-api/11768282#11768282
+extension UIResponder {
+    private weak static var firstResponder: UIResponder?
```

Comment:
The symbol `firstResponder` seems like it could collide with a symbol on UIResponder.  I wonder if we should rename this to `owsFirstResponder` or something?

Reply:
Swift won't let us collide without specifying `override`.